src/client/types.ts

```diff
@@ -304,8 +304,19 @@ type SecretPathMessage = {
   secretName: string;
 };
 
+type SecretPathsMessage = {
+  secretNames: Array<string>;
+};
+
```

Comment:
No point making a separate `SecretPathsMessage` if it's not used anywhere else. Just roll it into the `SecretRemoveMessage`